.github/workflows/release.yml

```diff
@@ -48,12 +48,26 @@ jobs:
           asset_path: ./target/release/xline
           asset_name: xline-linux-amd64
           asset_content_type: application/octet-stream
-
-      - name: Close the Relevant Milestone
-        id: close-milestone
-        uses: adlerhsieh/prepare-release@0.1.2
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          REPO_OWNER: 'datenlord'
-          REPO: 'Xline'
-          IGNORE_MILESTONE_NOT_FOUND: false
+      - run: |
+          mv ./target/release/xline ./scripts
+          mv ./target/release/benchmark ./scripts
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
I think qemu is not needed as we only build linux amd64 target.

Reply:
I think we can keep it in that we may support for other platforms.